src/lib/statistics/column_statistics.cpp

```diff
@@ -398,7 +398,11 @@ float ColumnStatistics<ColumnDataType>::estimate_range_selectivity(const ColumnD
   if (std::is_integral<ColumnDataType>::value) {
     return static_cast<float>(maximum - minimum + 1) / static_cast<float>(_max - _min + 1);
   } else {
-    return static_cast<float>(maximum - minimum) / static_cast<float>(_max - _min);
+    if (_max == _min) {
+      return 1.0f;
```

Comment:
Why not `return estimate_equals_with_value(_min)`?

Reply:
In case you mean `estimate_equals_with_value(minimum)` - that should do the same. Not any better, imo, I'll try it

Reply:
Ah, wait. I thought this was the `BETWEEN 5 and 5` case. Never mind.